.github/workflows/audit.yaml

```diff
@@ -0,0 +1,20 @@
+name: "Audit"
```

Comment:
We may want to hold this till after the release this week as there audit issues to resolve.